contracts/core/Core.sol

```diff
@@ -59,6 +65,15 @@ contract Core is
         _;
     }
 
+    // Checks to make sure a valid token address is given.
+    modifier isValidTokenAddress(address _tokenAddress) {
```

Comment:
I'm currently adding logic that allows checks to see whether the Set is Valid

Reply:
let me know when this is in and i can refactor

Reply:
this is the thing i was asking about @felix2feng